chat-plugins/wifi.js

```diff
@@ -224,9 +224,10 @@ var LotteryGiveAway = (function () {
 		for (var id in this.winners) {
 			finallist.push(this.winners[id].name);
 		}
+		var finallength = finallist.length;
```

Comment:
If you're only going to use this variable exactly once, why not just make it the string `'winner'` or `'winners'` here?